query/README.md

```diff
@@ -162,3 +164,10 @@ server.WithGateway(
   )
 )
 ```
+## Searching
+
+The syntax of REST representation of `infoblox.api.Searching` is the following.
+
+| Request Parameter | Description                              |
+| ----------------- |------------------------------------------|
+| _fts              | A string expression which performs a full-text-search on the DB |
```

Comment:
Is "full-text-search" a commonly understood term? I think it would be good to define what exactly it means, and possibly give an example or two